.aegir.js

```diff
@@ -28,14 +28,14 @@ export default {
         },
         peerId,
         transports: [
-          new WebSockets()
+          webSockets()
         ],
         streamMuxers: [
-          new Mplex()
+          mplex()
         ],
         connectionEncryption: [
-          new Noise(),
-          new Plaintext()
+          () => () => new Noise(),
```

Comment:
This is needed until https://github.com/ChainSafe/js-libp2p-noise/pull/229 is released